src/Control/HTTPRequest.php

```diff
@@ -468,6 +468,8 @@ public function offsetUnset($offset)
      * Construct an HTTPResponse that will deliver a file to the client.
      * Caution: Since it requires $fileData to be passed as binary data (no stream support),
      * it's only advisable to send small files through this method.
+     * this function needs to called inside the controllers response, eg
```

Comment:
```suggestion
     * This function needs to called inside the controller’s response, e.g.:
```

Reply:
This suggestion was resolved but not applied?

Reply:
I think it is resolved now.